dhash/module.py

```diff
@@ -383,9 +438,66 @@ async def _save_hashes(self, message: nextcord.Message):
             )
             yield h
 
+    async def _url_hashes(self, message: nextcord.Message):
+        for url in re.findall(URL_REGEX, message.content):
+            if self.allowed_urls and not re.search(self.allowed_urls, url):
+                continue
+            try:
+                req = urllib.request.Request(url, headers=HEADERS)
+                file = urllib.request.urlopen(req, timeout=20)
+            except (URLError, HTTPError, ValueError) as ex:
+                await guild_log.debug(
+                    message.author,
+                    message.channel,
+                    f"DHash could not check url {url}.",
+                    exception=ex,
```

Comment:
We may not need to save full traceback, error name/reason should be enough.

Reply:
There could be few reasons for this to fail, so IMHO it's better to know which function failed.

Reply:
Atleast for now, since there could be some edge cases hard to debug.

Reply:
The debugging should be made _before_ the PR gets merged. This way someone may spam your log with long invalid links that pretent to be images, for example. I don't think we need this.